Open Judge System/LocalWorker/OJS.LocalWorker/App.config

```diff
@@ -10,9 +10,9 @@
   </connectionStrings>
   <appSettings>
     <add key="MonitoringServiceExecutablePath" value="C:\OpenJudgeSystem\Open Judge System\LocalWorker\OJS.LocalWorkerMonitoring\bin\Debug\OJS.LocalWorkerMonitoring.exe" />
-    <add key="CSharpCompilerPath" value="C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\MSBuild\15.0\Bin\Roslyn\csc.exe" />
+    <add key="CSharpCompilerPath" value="C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin\Roslyn\csc.exe" />
```

Comment:
Should this stay? Should be changed on the server as well, so post deploy action label is needed...